server/controllers/request.controller.js

```diff
@@ -0,0 +1,61 @@
+const RequestInfo = require('../models/request.model.js');
+
+exports.getRequests = async (req, res) => {
+    try {
+        const requestInfos = await RequestInfo.find();
```

Comment:
This would be grabbing the requests for all users in the database even ones not pertaining to the authenticated user, which is a privacy concern and also a UX problem. I would filter by the requests that are relevant to the user.